src/Microsoft.IdentityModel.Protocols/Base64UrlConverter.cs

```diff
@@ -32,6 +32,9 @@
 
 namespace Microsoft.IdentityModel.Protocols
 {
+    /// <summary>
+    /// Converts an object to and from <see cref="Base64UrlConverter"/>.
```

Comment:
Are we referring to this class in the class description? We should describe what it does without referring to the class name. That will send people in loops.